src/Symfony/Component/DependencyInjection/Tests/Fixtures/php/services1-2.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
```

Comment:
There is [an issue on the php-cs-fixer](https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/1987) repository about using `/**` or `/*` for file header comments.

They finally make it configurable in the fixer. What position should we adopt here? Why using a file level docblock rather than a comment for this?